vimeo-networking/src/main/java/com/vimeo/networking/VimeoClient.java

```diff
@@ -80,7 +83,7 @@
  * @see <a href="https://developer.vimeo.com/api">The Vimeo API Docs</a>
  */
 @SuppressWarnings("unused")
-public class VimeoClient {
+final public class VimeoClient {
```

Comment:
Good suggestions. I will add them to my list of follow up items. Regarding your question, I doubt that there is a good reason for the difference, but I don't know for sure since I didn't write them. Regardless, the inconsistency is one of many reasons why the entire networking API needs to be refactored in a major way. It's a high priority for the near future of the library. I think that it is prudent to wait to address all of these issues at once with a complete coherent refactor.